test/unit/mock/files/crondotd

```diff
@@ -0,0 +1,10 @@
+#
```

Comment:
I think we need to provide a better mock file that has users in it. Here's the contents of the `/etc/cron.d/mdadm` file from the Ubuntu vagrant box I just spun up:

```
#
# cron.d/mdadm -- schedules periodic redundancy checks of MD devices
#
# Copyright © martin f. krafft <madduck@madduck.net>
# distributed under the terms of the Artistic Licence 2.0
#

# By default, run at 00:57 on every Sunday, but do nothing unless the day of
# the month is less than or equal to 7. Thus, only run on the first Sunday of
# each month. crontab(5) sucks, unfortunately, in this regard; therefore this
# hack (see #380425).
57 0 * * 0 root if [ -x /usr/share/mdadm/checkarray ] && [ $(date +\%d) -le 7 ]; then /usr/share/mdadm/checkarray --cron --all --idle --quiet; fi
```

Reply:
Thanks, my precise64 doesn't have this; even after I installed mdadm. =)